contexts/webkms-v1.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "AesKeyWrappingKey2019": "https://w3id.org/security#AesKeyWrappingKey2019",
```

Comment:
Same deal here... we should have a separate context for this suite.

Reply:
Extracted to https://github.com/digitalbazaar/aes-key-wrapping-2019-context/pull/1